gitalias.txt

```diff
@@ -505,7 +505,7 @@
 
   # Do everything we can to synchronize all changes
   get = !git pull --rebase && git submodule update --init --recursive
-  put = !git commit --all --message="$1" && git push
+  put = "!f() { git add -A && git commit -m \"$@\" && git push;  }; f"
```

Comment:
There is a big difference between `git commit --all` and `git add -A && git commit`. Is the change intentional?

Reply:
Excellent point, thank you.